<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Calculo</class>
 <widget class="QMainWindow" name="Calculo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>971</width>
    <height>376</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Promedio de calificaciones</string>
  </property>
  <property name="styleSheet">
   <string notr="true">font: 75 16pt &quot;Comic Sans MS&quot;;
background-color: rgb(112,82,255 );</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Comic Sans MS</family>
      <pointsize>16</pointsize>
      <weight>9</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 16pt &quot;Comic Sans MS&quot;;</string>
    </property>
    <property name="text">
     <string>Calificacion 1</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>20</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 16pt &quot;Comic Sans MS&quot;;</string>
    </property>
    <property name="text">
     <string>Calificacion 2</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>20</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 16pt &quot;Comic Sans MS&quot;;</string>
    </property>
    <property name="text">
     <string>Calificacion 3</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Calificacion 5</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>20</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Calificacion 4</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonCalc">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>160</y>
      <width>251</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
font: 87 14pt &quot;Arial Black&quot;;
color: rgb(255, 255, 255);
background-color: rgb(208,106,255 );
border :5px solid black;
border-radius: 20px;</string>
    </property>
    <property name="text">
     <string>Calcular</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="calif1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 16pt &quot;Comic Sans MS&quot;;
background-color: rgb(255, 255, 255);
border :5px solid black;
border-radius: 20px;</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="calif2">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>60</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 16pt &quot;Comic Sans MS&quot;;
background-color: rgb(255, 255, 255);
border :5px solid black;
border-radius: 20px;</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="calif3">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>60</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 16pt &quot;Comic Sans MS&quot;;
background-color: rgb(255, 255, 255);
border :5px solid black;
border-radius: 20px;</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="calif4">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>60</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 16pt &quot;Comic Sans MS&quot;;
background-color: rgb(255, 255, 255);
border :5px solid black;
border-radius: 20px;</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="calif5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>170</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 16pt &quot;Comic Sans MS&quot;;
background-color: rgb(255, 255, 255);
border :5px solid black;
border-radius: 20px;</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QTextEdit" name="resultado">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>270</y>
      <width>621</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Comic Sans MS</family>
      <pointsize>16</pointsize>
      <weight>9</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
border :5px solid black;
border-radius: 20px;</string>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
